### DISCUSSION TITLE
Not support Chinese link anchor jump on Android

### DISCUSSION THREAD

**Discussion by adoyle-h** [community/community#48984] [Mobile] [answered]
### Select Topic Area

Bug

### Body

For example, https://github.com/just-talks/tech-talks#toc
Click the link of Chinese text in TOC. It will jump to the position in README. It works on PC. But it doesn't work on Github Mobile App.

My Github Mobile version: v1.101.0 (706)

**Comment by eliperkins** [comment, marked as answer]
Thanks for pointing this bug out, @adoyle-h! I'm able to replicate this on my end, so I've filed an issue on our internal tracking repo to get this bug fixed. I'll keep you posted!

**Reply by adoyle-h** [reply]
This issue still exist with Github Mobile v1.103.0 (708).
